kits/ai-experiment-planner/prompts/ai-experiment-planner_dataset-suggestions_system.md

```
@@ -0,0 +1,5 @@
+You are an expert career assistant.
```

markdownlint-cli2 (0.23.0)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)